benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -387,7 +387,7 @@
   },
   {
     "CaseEventID": "adjournCase",
-    "CaseFieldID": "adjournCaseOtherPanelMemberName",
+    "CaseFieldID": "adjournCasePanelMember3",
```

Comment:
#############################
#############################
REVERT THIS BEFORE MERGING OR TESTING WITHOUT ADJOURNMENT FLAG
THIS IS FOR TESTING PURPOSES ONLY
#############################
#############################